### DISCUSSION TITLE
How do I change composers PHP version?

### DISCUSSION THREAD

**Discussion by JenuelDev** [hjbdev/pvm#22] [General]
Is the PVM also automatically sets the composers PHP version?

**Comment by hjbdev** [comment]
pvm won't automatically set composer's PHP version. If you're installing composer on Windows it will often set the PATH environment variable to a fixed version of PHP, which might interfere with pvm. It's best to remove the fixed version of php in your PATH, only having the entry that points to `<your user folder>\.pvm\bin`

**Reply by JenuelDev** [reply]
I see it now, when installing composer in windows, select the developer mode.
